plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -1257,6 +1257,34 @@ public void setTableComment(ConnectorSession session, ConnectorTableHandle table
         metastore.commentTable(handle.getSchemaName(), handle.getTableName(), comment);
     }
 
+    @Override
+    public void setViewComment(ConnectorSession session, SchemaTableName viewName, Optional<String> comment)
+    {
+        Table view = metastore.getTable(viewName.getSchemaName(), viewName.getTableName())
```

Comment:
Since view information is store in `Table`, is there is any reason on why we don't use `metastore#commentTable` ?

Reply:
A view comment is stored as base64 format in metastore's `VIEW_ORIGINAL_TEXT` column like below as you may already know. 
```
/* Presto View: eyJvcmlnaW5hbFNxbCI6IlNFTEVDVCAxIGMxXG5cbiIsImNhdGFsb2ciOiJoaXZlIiwic2NoZW1hIjoiZGVmYXVsdCIsImNvbHVtbnMiOlt7Im5hbWUiOiJjMSIsInR5cGUiOiJpbnRlZ2VyIn1dLCJjb21tZW50IjoidGVzdCBjb21tZW50Iiwib3duZXIiOiJ5dXlhLmViaWhhcmEiLCJydW5Bc0ludm9rZXIiOmZhbHNlfQ== */
```

```json
{
    "originalSql": "SELECT 1 c1\n\n",
    "catalog": "hive",
    "schema": "default",
    "columns": [
        {
            "name": "c1",
            "type": "integer"
        }
    ],
    "comment": "test comment",
    "owner": "yuya.ebihara",
    "runAsInvoker": false
}
```